index.html

```diff
@@ -1,459 +1,451 @@
+          otherLinks: [{
+              key: "Related Documents",
+              data: [{
+                  value: "Verifiable Credentials Data Model v2.0",
+                  href: "https://www.w3.org/TR/vc-data-model-2.0/"
+              }, {
+                  value: "Controller Documents 1.0",
+                  href: "https://www.w3.org/TR/controller-document/"
+              }]
+          }],
+          localBiblio: {
+              "SD-JWT": {
+                  title: "Selective Disclosure for JWTs (SD-JWT)",
+                  href: "https://datatracker.ietf.org/doc/html/draft-ietf-oauth-selective-disclosure-jwt",
+                  authors: ["Daniel Fett", "Kristina Yasuda", "Brian Campbell"],
+                  status: "Internet-Draft",
+                  publisher: "IETF"
+              },
+              "SD-JWT-VC": {
+                  title: "SD-JWT-based Verifiable Credentials (SD-JWT VC)",
+                  href: "https://datatracker.ietf.org/doc/html/draft-ietf-oauth-sd-jwt-vc",
+                  authors: ["Oliver Terbu", "Daniel Fett", "Brian Campbell"],
+                  status: "Internet-Draft",
+                  publisher: "IETF"
+              },
+              "JOSE-REGISTRIES": {
+                  title: "The JSON Object Signing and Encryption (JOSE) Registries",
+                  href: "https://www.iana.org/assignments/jose",
+                  authors: ["The Internet Assigned Numbers Authority"],
+                  status: "REC",
+                  publisher: "The Internet Assigned Numbers Authority"
+              },
+              "SP-800-122": {
+                  title: "Guide to Protecting the Confidentiality of Personally Identifiable Information (PII)",
+                  href: "https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication800-122.pdf",
+                  authors: ["Erika McCallister", "Tim Grance", "Karen Scarfone"],
+                  status: "Special Publication 800-122",
+                  publisher: "NIST"
+              },
+          }
+      };
     </script>
-</head>
-
-<body>
-<section id="abstract">
-    <p>
+  </head>
+  <body>
+    <section id="abstract">
+      <p>
         This specification defines how to secure credentials and presentations
         conforming to the Verifiable Credential data model [[VC-DATA-MODEL-2.0]]
-        with JSON Object Signing and Encryption (<a href="https://datatracker.ietf.org/wg/jose/about/">JOSE</a>),
-        Selective Disclosure for JWTs [[SD-JWT]],
-        and CBOR Object Signing and Encryption (COSE) [[RFC9052]].
-        This enables the Verifiable Credential data model
-        [[VC-DATA-MODEL-2.0]] to be implemented with standards
+        with JSON Object Signing and Encryption 
+        (<a href="https://datatracker.ietf.org/wg/jose/about/">JOSE</a>),
+        Selective Disclosure for JWTs [[SD-JWT]], and CBOR Object Signing and 
+        Encryption (COSE) [[RFC9052]]. This enables the Verifiable Credential 
```

Comment:
"CBOR Object Signing and Encryption" is broken across lines, when it's a term that should be searchable and so needs to be on a single line.